core/variant/variant.cpp

```diff
@@ -1700,11 +1700,13 @@ String Variant::stringify(int recursion_count) const {
 		case DICTIONARY: {
 			const Dictionary &d = *reinterpret_cast<const Dictionary *>(_data._mem);
 			if (recursion_count > MAX_RECURSION) {
-				ERR_PRINT("Max recursion reached");
-				return "{...}";
+				ERR_PRINT("Maximum dictionary recursion reached!");
+				return "{ ... }";
 			}
 
-			String str("{");
+			// Add leading and trailing space to Dictionary printing. This distinguishes it
+			// from array printing on fonts that have similar-looking {} and [] characters.
+			String str("{ ");
```

Comment:
Only change I'm not 100% about. Leads to an "asymmetry" of sorts.

Reply:
It's common for JavaScript formatters to format single-line arrays like this `[1, 2]`, but objects like this `{ hello: "world" }`.

Reply:
I see what you mean! 
![javascript console logging two values, an empty object and an empty array. the object shows spaces between the curly braces](https://user-images.githubusercontent.com/14253836/180672544-b486083c-3190-40a9-9f2d-465f6b5fc1be.png)